src/components/LabeledTags.vue

```diff
@@ -48,10 +48,10 @@ export default defineComponent({
 .labeled-tag {
   background-color: $color-alto;
   border: 1px solid $color-white;
-  border-radius: 10px;
-  margin-right: 0.25rem;
+  border-radius: 5px;
+  margin-right: 0.12rem;
```

Comment:
I know this was the requested value, but it is translated to `1.920px` - Instead of dealing with fractional pixels I'd rather we round the given values and use them as such

```suggestion
  margin-right: 2px;
```

Reply:
Done